In [1]:
data = [1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]

In [2]:
data

[1.13,
 1.55,
 1.43,
 0.92,
 1.25,
 1.36,
 1.32,
 0.85,
 1.07,
 1.48,
 1.2,
 1.33,
 1.18,
 1.22,
 1.29]

In [3]:
!pip install scipy

In [9]:
import scipy
from scipy import stats
import pandas as pd
import numpy as np

In [10]:
data = pd.DataFrame(data)

In [11]:
data

0
0   1.13
1   1.55
2   1.43
3   0.92
4   1.25
5   1.36
6   1.32
7   0.85
8   1.07
9   1.48
10  1.20
11  1.33
12  1.18
13  1.22
14  1.29

In [12]:
data.describe()      #for mean and std

0
count  15.000000
mean    1.238667
std     0.193164
min     0.850000
25%     1.155000
50%     1.250000
75%     1.345000
max     1.550000

Mean of our data = 1.238667

a. Build 99% Confidence Interval Using Sample Standard Deviation
-  We have to find probability for the mean number(1.238667 Million) of characters printed before the print-head fails with 99% Confidence Interval

-  Our mean is a constant value and not a range and so the probability for mean no. of characters printed before print-head fails with 99% confidence Interval is  0.

b. Build 99% Confidence Interval Using Known Population Standard Deviation

-  same as above, our mean is a constant value and not a range and so the probability for mean no. of characters printed before print-head fails with 99% confidence Interval is 0.